### DISCUSSION TITLE
Adding Danish version of Archinstall

### DISCUSSION THREAD

**Discussion by COHL7500** [archlinux/archinstall#1329] [Q&A] [answered]
Good day! 

I would like to contribute to the project by translating the installation process to danish.

How do I commence and edit the text for the installer? 

Thanks in advance.

**Comment by Torxed** [comment, marked as answer]
Hi, happy to hear!

How much of the contribution documentation for locales have you read? https://github.com/archlinux/archinstall/blob/master/archinstall/locales/README.md

If anything is unclear we're happy to help!

**Reply by COHL7500** [reply]
I managed to find the linked documentation yesterday and I already translated 20%. 

Thanks for the help :) Have a nice day

**Reply by Torxed** [reply]
Awesome to hear :D We can probably link to it from the main README to make it a bit easier to find :)

Ser frem til din oversættelse :)

**Reply by COHL7500** [reply]
I'm now done with the translation. However, I have not done any testing, though I've checked through for spelling errors/typos and it seems good to go. 

Should I just add a pull request or is there any prerequisite? Thanks in advance

**Reply by Torxed** [reply]
A straight up PR is welcome. I'll be back in a few days to merge things :)
